docs/guides/esp32/build_app_and_commission.md

```diff
@@ -86,21 +87,25 @@ functionality can still work fine.
 
     All the example applications supports target chips: esp32, esp32s3, esp32c3
 
-    ESP32H2 is only supported in lighting-app, to set it as target
+    ESP32H2 and ESP32C6 are only supported in lighting-app, to set it as target
 
     ```
-    idf.py --preview set-target esp32h2
+    $ idf.py --preview set-target esp32h2
+    $ idf.py --preview set-target esp32c6
     ```
 
 -   Configuration Options
 
     To build the default configuration (`sdkconfig.defaults`) skip this step.
 
-    To build a specific configuration (example `m5stack`):
+    To build a specific configuration (example `m5stack` or `esp32h2` or
+    `esp32c6`):
 
     ```
     $ rm sdkconfig
     $ idf.py -D 'SDKCONFIG_DEFAULTS=sdkconfig_m5stack.defaults' build
+    $ idf.py -D 'SDKCONFIG_DEFAULTS=sdkconfig.defaults.esp32h2' build
```

Comment:
`-D 'SDKCONFIG_DEFAULTS=sdkconfig.defaults.<target_name>'` is not required if the sdkconfig file name has a target_name suffix, because when we set target, the `sdkconfig.defaults.<target_name>` will be applied with the `sdkconfig.defaults`.